src/main/java/mServer/crawler/sender/arte/ArteJsonObjectToDatenFilmCallable.java

```diff
@@ -125,27 +125,6 @@ public DatenFilm call() {
         return film;
     }
     
-    
-    private LocalDateTime readDateTime(String aFilmDetails) throws IOException
-    {
-        DateTimeFormatter dateTimeFormatter = new DateTimeFormatterBuilder().appendPattern("EEEE', 'd'. 'MMMM' um 'H.mm' Uhr'")
-            .parseDefaulting(ChronoField.HOUR_OF_DAY, 0)
-            .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
-            .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0)
-            .parseDefaulting(ChronoField.YEAR, 2017)
-            .toFormatter(Locale.GERMANY); 
-            Matcher matcher = Pattern.compile(REGEX_PATTERN_DATETIME).matcher(aFilmDetails);
-        if(matcher.find())
-        {
-            LocalDateTime localDateTime = LocalDateTime.parse(matcher.group().trim().replaceAll("\\n", "").replaceAll("\\s{2,}", " "),dateTimeFormatter);
-            return localDateTime;
-        } else {
-            return LocalDateTime.of(LocalDate.now(),LocalTime.MIDNIGHT);
-        }
-        
-        
-    }
-    
     private LocalTime durationAsTime(long aDurationInSeconds) throws IOException
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)